pims/image_sequence.py

```diff
@@ -102,6 +121,17 @@ def _get_files(self, path_spec):
             self._count = len(path_spec)
             return
 
+        if zipfile.is_zipfile(path_spec):
+            self.is_zipfile = True
```

Comment:
Can you put either an exception or a warning in here if the user passes both a zipfile _and_ a plugin?  Silently ignoring user input is a bad thing.